README.md

```diff
@@ -39,11 +39,11 @@ For new apps created with `dart create`, the lints are enabled by default.
 
 For existing apps or packages, take these steps to enable these lints:
 
-1. Add a *dev* dependency on this package to your `pubspec.yaml` file:
+1. Add a *dev* dependency on this package to your `pubspec.yaml` file.  At the root
+of your package:
```

Comment:
Maybe "In a terminal, located at the root of your package"?

Reply:
Done.